mindee/documents/financial/financial_v1.py

```diff
@@ -1,52 +1,64 @@
 from typing import List, Optional, TypeVar
 
 from mindee.documents.base import Document, TypeApiPrediction, clean_out_string
-from mindee.documents.invoice.invoice_v3 import InvoiceV3
-from mindee.documents.receipt.receipt_v3 import ReceiptV3
-from mindee.endpoints import Endpoint
+from mindee.documents.financial.line_item import InvoiceLineItem
 from mindee.fields.amount import AmountField
 from mindee.fields.company_registration import CompanyRegistrationField
 from mindee.fields.date import DateField
 from mindee.fields.locale import LocaleField
 from mindee.fields.payment_details import PaymentDetails
 from mindee.fields.tax import TaxField
 from mindee.fields.text import TextField
-from mindee.input.sources import InputSource
 
 
 class FinancialV1(Document):
     locale: LocaleField
     """locale information"""
-    total_incl: AmountField
+    total_amount: AmountField
     """Total including taxes"""
-    total_excl: AmountField
+    total_net: AmountField
     """Total excluding taxes"""
     date: DateField
-    """Date the document was issued"""
-    time: TextField
-    """Time the document was issued"""
+    """Date the invoice was issued"""
     invoice_number: TextField
     """Invoice number"""
+    reference_numbers: List[TextField]
+    """List of Reference numbers including PO number."""
     due_date: DateField
     """Date the invoice is due"""
-    taxes: List[TaxField]
+    taxes: List[TaxField] = []
     """List of all taxes"""
-    merchant_name: TextField
-    """Merchant/Supplier's name"""
+    total_tax: AmountField
+    """Sum total of all taxes"""
+    supplier_name: TextField
+    """Supplier 's name"""
     supplier_address: TextField
-    """Merchant/Supplier's address"""
+    """Supplier's address"""
+    supplier_company_registrations: List[CompanyRegistrationField]
+    """Company numbers"""
     customer_name: TextField
     """Customer's name"""
     customer_address: TextField
     """Customer's address"""
-    customer_company_registration: List[CompanyRegistrationField]
+    customer_company_registrations: List[CompanyRegistrationField]
     """Customer company registration numbers"""
-    payment_details: List[PaymentDetails]
+    supplier_payment_details: List[PaymentDetails]
     """Payment details"""
-    company_number: List[CompanyRegistrationField]
-    """Company numbers"""
-    total_tax: AmountField
-    """Sum total of all taxes"""
+    line_items: List[InvoiceLineItem]
+    """Details of line items"""
+    tip: AmountField
+    """Total amount of tip and gratuity."""
+    time: TextField
+    """Time as seen on the receipt in HH:MM format."""
+    document_type: TextField
+    """
+    A classification field, that can return 4 values:
+    `EXPENSE RECEIPT`, `CREDIT CARD RECEIPT`, `INVOICE`, `CREDIT NOTE`
```

Comment:
I think that we should not list all the available types here.
Why ?
Because if the list change from the API source, the information will be incomplete and we would need to update this documentation.
If you really want to provide information, I instead suggest to put the link to the official documentation (https://developers.mindee.com/docs/financial-documents-ocr#document-type)

Reply:
done